src/containers/NFT/NFTHeader/Details.tsx

```diff
@@ -13,12 +13,12 @@ interface Props {
   data: NFTFormattedInfo & AccountFormattedInfo & MintedProps
 }
 
-const Details = ({ data }: Props) => {
+export const Details = ({ data }: Props) => {
   const { minted, domain, NFTTaxon, uri, transferFee } = data
   const { t } = useTranslation()
   const language = useLanguage()
   const formattedFee =
-    transferFee &&
+    transferFee !== undefined &&
```

Comment:
i prefer `transferFee != null` but nbd

Reply:
reverted back to the original, explicitly check for null or undefined isnt needed here